Automattic/newspack-popups
# Fix editor colors

### All Submissions:






### Changes proposed in this Pull Request:



Fixes popup color issues in the editor.

### How to test the changes in this Pull Request:

1. On `master`, 
2. Create a new Campaign, fill in a title, and set a campaign background color to black
2. Observe the background color is not applied, and the title color is white (which makes it invisible)
3. Switch to this branch and rebuild
4. Redo step 2
5. Observe the background and text colors are applied as expected
6. Close the sidebar color panel and reload the editor, observe that colors are applied without the sidebar panel being open

### Other information:
